src/main/java/de/retest/recheck/cli/subcommands/Ignore.java

```diff
@@ -73,7 +74,7 @@ public void run() {
 				}
 				TestReportUtil.print( report, testReport );
```

Comment:
Which report do we want to print? The original or the filtered so that one can see what differences are going to be ignored?

Reply:
We want to print the given report